src/lib/form-field/form-field.scss

```diff
@@ -45,6 +45,9 @@ $mat-form-field-default-infix-width: 180px !default;
   white-space: nowrap;
   flex: none;
   position: relative;
+
+  // Allow for projected block-level elements not stand next to each other, rather than stack.
+  display: flex;
```

Comment:
@mmalerba as far as I could tell there shouldn't be any visual differences, but you might have some more insights.